src/templates/index.html

```diff
@@ -25,27 +25,15 @@
 og_sitename: Kottans
 fb_admins: 100003342204377, 1744734572, 100000443393783, 100001674114934
 ---
-{% extends "layouts/_layout.html" %}
 
-{% block content %}
-
-{% include "partials/_header.html" %}
+{% extends "layouts/_layout.html" %} {% block content %} {% include
+"partials/_header.html" %}
```

Comment:
As for me, it must be more readable.

Reply:
Pls, see second checkbox in the description of PR.